composer.json

```diff
@@ -10,10 +10,16 @@
     ],
     "require": {
         "php": "^7.1",
-        "phpspec/prophecy": "^1.7",
-        "phpunit/phpunit": "^6.0||^7.0",
         "phpstan/phpstan": "^0.10"
     },
+    "require-dev": {
+        "phpspec/prophecy": "^1.7",
+        "phpunit/phpunit": "^6.0||^7.0"
+    },
+    "conflicts": {
+        "phpspec/prophecy": "<1.7, >=2.0",
+        "phpunit/phpunit": "<6.0"
```

Comment:
Sorry for being pedanting, but maybe we should do the same here, conflicting with `<6, >=8`